### DISCUSSION TITLE
request is not passed to filterset

### DISCUSSION THREAD

**Discussion by lubaskinc0de** [carltongibson/django-filter#1497] [Q&A] [answered]
i am passing request.GET to views but request is not being passed to getattr which is causing error 'NoneType' object has no attribute 'user'
![Снимок экрана (86)](https://user-images.githubusercontent.com/100635212/164972665-c82d3c61-b07e-4b84-a4b4-5eaf885356cd.png)

**Comment by aliceni81** [comment]
Encounter same problem

**Comment by aliceni81** [comment, marked as answer]
@lubaskinc0de I found the solution.
Need to pass request to the filter.
e.g. RequestFilter(request.GET, queryset=requests, request=request)

**Reply by lubaskinc0de** [reply]
thx you!
